src/stylesheets/elements/_inputs.scss

```diff
@@ -222,79 +220,79 @@ legend {
   width: 1.4rem;
 }
 
-[type=checkbox]:checked + label::before,
-[type=radio]:checked + label::before {
+.usa-checkbox-input:checked + .usa-checkbox-label::before,
+.usa-radio-input:checked + .usa-radio-label::before {
   background-color: $color-primary;
   box-shadow: 0 0 0 1px $color-primary;
 }
 
-[type=radio]:checked + label::before {
+.usa-radio-input:checked + .usa-radio-label::before {
   box-shadow: 0 0 0 2px $color-white, 0 0 0 4px $color-primary;
 }
 
-[type=checkbox]:checked + label::before,
-[type=checkbox]:checked:disabled + label::before {
+.usa-checkbox-input:checked + .usa-checkbox-label::before,
+.usa-checkbox-input:checked:disabled + .usa-checkbox-label::before {
   background-image: url('#{$theme-image-path}/correct8.png');
   background-image: url('#{$theme-image-path}/correct8.svg');
   background-position: 50%;
   background-repeat: no-repeat;
 }
 
-[type=radio]:focus + label::before {
+.usa-radio-input:focus + .usa-radio-label::before {
   outline: $focus-outline;
-  outline-offset: $focus-spacing * 2; // Double the offset to account for circular shape
+  outline-offset: units(0.5);
 }
 
-[type=checkbox]:disabled + label {
+.usa-checkbox-input:disabled + .usa-checkbox-label {
   color: $color-gray-lighter;
 }
 
-[type=checkbox]:focus + label::before {
+.usa-checkbox-input:focus + .usa-checkbox-label::before {
   outline: $focus-outline;
   outline-offset: $focus-spacing;
 }
 
-[type=checkbox]:disabled + label::before,
-[type=radio]:disabled + label::before {
+.usa-checkbox-input:disabled + .usa-checkbox-label::before,
+.usa-radio-input:disabled + .usa-radio-label::before {
   background: $color-gray-lightest;
   box-shadow: 0 0 0 1px $color-gray-light;
   cursor: not-allowed;
 }
 
 // Range inputs
 
-[type=range] {
+.usa-range {
   appearance: none;
   border: none;
   padding-left: 0;
   width: 100%;
 }
 
-[type=range]::-webkit-slider-runnable-track {
+.usa-range::-webkit-slider-runnable-track {
   background: $color-gray-light;
   border: 1px solid $color-gray-medium;
   cursor: pointer;
   height: 1.2rem;
   width: 100%;
 }
 
-[type=range]::-moz-range-track {
+.usa-range::-moz-range-track {
   background: $color-primary;
   border: 1px solid $color-gray-medium;
   cursor: pointer;
   height: 1.2rem;
   width: 100%;
 }
 
-[type=range]::-ms-track {
+.usa-range::-ms-track {
   background: transparent;
   color: transparent;
   cursor: pointer;
   height: 1.2rem;
   width: 100%;
 }
 
-[type=range]::-webkit-slider-thumb {
+.usa-range::-webkit-slider-thumb {
   -webkit-appearance: none;
```

Comment:
Again, unrelated to this PR, but should the webkit-thumb explicitly have the same bg color as the other two?